articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to create a Availability Set | UKCloud Ltd
+description: Creating a Availability Set in UKCloud for Microsoft Azure
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: 
+toc_sub3:
+toc_sub4:
+toc_title: Create a Availability
+toc_fullpath: Users/How To/azs-how-create-availability-set.md
+toc_mdlink: azs-how-create-availability-set.md
+---
+
```

Comment:
General comment: It's not a good idea to lift text straight from the Microsoft docs as this could be considered plagiarism or even copyright infringement. Try to use your own words.

Reply:
rewritten certain aspects to ensure it doesn't read as if its just lifted